_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_vaapi.cpp

```diff
@@ -1248,17 +1248,17 @@ bool operator!=(const ENCODE_ENC_CTRL_CAPS& l, const ENCODE_ENC_CTRL_CAPS& r)
     return !(l == r);
 }
 
-
-void CUQPMap::Init (mfxU32 picWidthInLumaSamples, mfxU32 picHeightInLumaSamples)
+// block width/height = 8 << blockSize
+void CUQPMap::Init (mfxU32 picWidthInLumaSamples, mfxU32 picHeightInLumaSamples, mfxU32 blockSize)
```

Comment:
Do we begin supporting any block size? AFAIR, HEVC MBQP could work only with 16x16 blocks. It's captured in the manual. Do we need to update manual ?

Reply:
Now only 16 and 32 are used by hw. From msdk code point of view any size 8<<N can work.

Reply:
I was wrong, 16x16 is not captured in the manual:
https://github.com/Intel-Media-SDK/MediaSDK/blob/master/doc/mediasdk-man.md#mfxExtMBQP
But I think it's worth mentioning in the manual that encoder can work with different block sizes and describe what API variable can set it. Leonid, can you please do it?

Reply:
Oh no. This commit doesn't change anything related to mfxExtMBQP. It aligns msdk with driver/hw.